receiver/windowsservicereceiver/scraper.go

```diff
@@ -0,0 +1,25 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package windowsservicereceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/windowsservicereceiver"
+
+import (
+	"context"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/receiver"
+)
+
+type windowsServiceScraper struct {
+	cfg      *Config
+	settings *receiver.Settings
+}
+
+func (wss *windowsServiceScraper) Start(ctx context.Context, _ component.Host) error {
+
+	return nil
```

Comment:
Could you please add a TODO comment where appropriate? (same apply to other locations)

I expect opening of the SCM to happen at the start of the receiver. Keep in mind that `Shutdown` must work even if `Start` was not called.